src/publisher/management/commands/ingest.py

```diff
@@ -164,7 +163,12 @@ def handle_many(print_queue, action, path, force, dry_run):
         LOG.info("found no article json at %r" % os.path.abspath(path))
         clean_up(print_queue)
         return
-    Parallel(n_jobs=-1)(delayed(job)(print_queue, action, path, force, dry_run) for path in ajson_file_list) # pylint: disable=unexpected-keyword-arg
+    try:
+        aws_events.notify(STOP) # defer sending notifications
+        Parallel(n_jobs=-1)(delayed(job)(print_queue, action, path, force, dry_run) for path in ajson_file_list) # pylint: disable=unexpected-keyword-arg
```

Comment:
ah, that's why the concurrent queue; is the multiple job version used only during backfill or are there other scenarios?